FeatureDetect/src/external/CMakeLists.txt

```diff
@@ -0,0 +1 @@
+add_subdirectory(opencv)
```

Comment:
Let's add a trailing newline for consistency

Reply:
done